Assets/MRTK/Examples/Demos/HandTracking/Scripts/TetheredPlacement.cs

```diff
@@ -11,26 +12,35 @@ namespace Microsoft.MixedReality.Toolkit.Examples.Demos
     [AddComponentMenu("Scripts/MRTK/Examples/TetheredPlacement")]
     public class TetheredPlacement : MonoBehaviour
     {
-        [Tooltip("The distance from the GameObject's spawn position at which will trigger a respawn ")]
-        public float DistanceThreshold = 20.0f;
+        [SerializeField, Tooltip("The distance from the GameObject's spawn position at which will trigger a respawn."), FormerlySerializedAs("DistanceThreshold")]
+        private float distanceThreshold = 20.0f;
 
-        private Vector3 RespawnPoint;
-        private Quaternion RespawnOrientation;
+        /// <summary>
+        /// The distance from the GameObject's spawn position at which will trigger a respawn.
+        /// </summary>
+        public float DistanceThreshold
+        {
+            get => distanceThreshold;
+            set => distanceThreshold = value;
+        }
 
+        private Vector3 localRespawnPosition;
+        private Quaternion localRespawnRotation;
         private Rigidbody rigidBody;
+        private float distanceThresholdSquared;
 
         private void Start()
         {
             rigidBody = GetComponent<Rigidbody>();
-
             LockSpawnPoint();
+            distanceThresholdSquared = distanceThreshold * distanceThreshold;
         }
 
         private void LateUpdate()
         {
-            float distanceSqr = (RespawnPoint - this.transform.position).sqrMagnitude;
+            float distanceSqr = (localRespawnPosition - transform.localPosition).sqrMagnitude;
 
-            if (distanceSqr > DistanceThreshold * DistanceThreshold)
+            if (distanceSqr > distanceThresholdSquared)
```

Comment:
With this change, updates to DistanceThreshold do not get applied (distanceThresholdSquared is only calculated in Start)

Reply:
Updated!